metafacture-xml/src/main/java/org/metafacture/xml/SimpleXmlEncoder.java

```diff
@@ -218,6 +216,25 @@ public void setNamespaces(final Map<String, String> namespaces) {
         this.namespaces = namespaces;
     }
 
+    /**
+     * Sets the namespace(s).
+     *
+     * @param namespacesString the namespaces as a String. It allows Java Properties
+     *                         structure, i.e. a key-value structure where the key is separated from the value
+     *                         by an equal sign '=', a semicolon ':' or a white space ' '.Multiple namespaces
```

Comment:
`:` is "colon", not "semicolon"; and space missing after period.

```suggestion
     *                         by an equal sign '=', a colon ':' or a white space ' '. Multiple namespaces
```